### ISSUE TITLE
Do not assume inputs to be inside labels

### ISSUE THREAD

**Issue by rixx** [zostera/django-bootstrap4#181]
I use `django-bootstrap4` with a custom widget. When upgrading to 1.0.x,  [this line](https://github.com/zostera/django-bootstrap4/blob/master/src/bootstrap4/renderers.py#L336):

```python
            label.input.attrs["class"] = label.input.attrs.get("class", []) + ["form-check-input"]
 ```

threw uncontrolled errors, because at that point, ``label.input`` does not exist. While Django in general has the behaviour of wrapping labels around inputs, not all widgets will follow this (non-standard) convention.

As far as I can tell, the correct way to determine the input element in question would be:

```python
            input_id = label.attrs.get("for")
            if input_id:
                label_input = soup.find(id=input_id)
                if label_input:
                    label_input.attrs["class"] = label_input.attrs.get("class", []) + ["form-check-input"]
```

This is a crash-bug preventing an update to django-bootstrap4 for me.

**Comment by dyve** [comment]
Could you share the html generated by this custom widget, so we can better understand this use case?

**Comment by rixx** [comment]
Sure. It looks like this:

```html
<div id="id_locales">
    <div class="checkbox">
        <input type="checkbox" name="locales" value="en" class="" title="" id="id_locales_0" checked="" lang="en">
        <label for="id_locales_0">English</label>
    </div>
    <div class="checkbox">
        <input type="checkbox" name="locales" value="de" class="" title="" id="id_locales_1" lang="de">
        <label for="id_locales_1">German</label>
    </div>
   <div class="checkbox">
        <input type="checkbox" name="locales" value="fr" class="" title="" id="id_locales_2" lang="fr">
        <label for="id_locales_2">French</label>
    </div>
</div>
```

**Comment by dyve** [comment]
Fixed in v1.1.0
